src/components/selectionBoxLayer.ts

```diff
@@ -151,7 +191,8 @@ export module Components {
       }
       this._xScale = xScale;
       this._xScale.onUpdate(this._adjustBoundsCallback);
-      this._bindBoxDataValues();
+      this._syncPixelPositionAndValue();
+      this.render();
```

Comment:
Now you will call `_syncPixelPositionAndValue()` twice. Once here and once inside the `renderImmediately()`.